.gitignore

```diff
@@ -1,5 +1,5 @@
 # Cargo stuff
-/target
+target
```

Comment:
We're not only writing Rust in this repository, so we really do want to ignore just the top-level `target` directory here (it's possible we could introduce some other `target` directory elsewhere that we do want in VC, and then we'll just have to remove this again). Also, we typically prefer to keep PRs focused on one thing, without sprinkling in other miscellaneous changes when we don't have to.